tests/test_actions.py

```diff
@@ -259,7 +273,7 @@ def test_raises_without_key(self, Tester):
     def test_adds_note_to_director(self, Tester):
         key = "key"
         value = "note"
-        MockQuestion = mock.Mock()
+        MockQuestion = mock.create_autospec(Question, instance=True)
```

Comment:
Should we have a `get_mock_question` helper function like the `get_mock_action` helper above?

Reply:
It crossed my mind as I was editing.  I don't think a helper function here would be problematic. It would make maintenance a little easier but at very slight cost of reading the test. I lean more towards explicit rather than easier to maintain in unittests.  The number of lines of code wouldn't be reduced.